pkg/kritis/apis/kritis/v1beta1/attestationauthority.go

```diff
@@ -33,9 +39,37 @@ type AttestationAuthority struct {
 
 // AttestationAuthoritySpec is the spec for a AttestationAuthority resource
 type AttestationAuthoritySpec struct {
-	NoteReference string   `json:"noteReference"`
-	PublicKeyList []string `json:"publicKeyList"`
-	PolicyType    string   `json:"policyType"`
+	NoteReference string      `json:"noteReference"`
+	PublicKeys    []PublicKey `json:"publicKeys"`
+	PolicyType    string      `json:"policyType"`
+}
+
+// PublicKey stores key data used to verify Attestations.
+type PublicKey struct {
+	// KeyId is the ID of this public key (required for PKIX keys). For PKIX
+	// keys, please use a RFC3986 conformant URI. For PGP keys, this field
+	// should be the OpenPGP RFC4880 V4 fingerprint of the key or be left
+	// blank.
```

Comment:
Please document how blank is interpreted.

Reply:
Done.